NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -1430,6 +1431,7 @@ public bool ExportToPDF(string path, string? password)
     {
         try
         {
+            Directory.CreateDirectory(System.IO.Path.GetDirectoryName(path));
```

Comment:
Should this also be added to ExportToCSV Function?

Reply:
It is added (line 1414)